nevergrad/functions/cycling/MensTeamPursuit.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+
+from .TeamPursuit import TeamPursuit
+from .Cyclist import Cyclist
+from .SimulationResult import SimulationResult
+import sys
+import math
+
+class MensTeamPursuit(TeamPursuit):
+
+	TEAM_SIZE = 4
+	RACE_DISTANCE = 4000
+	LAP_DISTANCE = 250
+	RACE_SEGMENTS = int((2 * (RACE_DISTANCE / LAP_DISTANCE)) - 1)
+	MAXIMUM_TRANSITIONS = RACE_SEGMENTS - 1
+
+	def __init__(self):
+		super().updateAirDensity()
+		self.team = []
+		for i in range(0, self.TEAM_SIZE):
+			self.team.append(Cyclist(1.75, 75.0, 6.0, self, i+1, "male"))
+
+	def simulate(self, transitionStrategy, pacingStrategy):
+		
+		if (len(transitionStrategy) != self.MAXIMUM_TRANSITIONS):
+			sys.exit("Transition strategy for the mens team pursuit must have exactly " + str(self.MAXIMUM_TRANSITIONS) +" elements")
```

Comment:
Please raise errors instead of exiting, and you can also use f-string for formatting string, they are very coder-friendly:
```suggestion
			raise ValueError(f"Transition strategy for the mens team pursuit must have exactly {self.MAXIMUM_TRANSITIONS} elements")
```